templates/worker/00-worker/baremetal/files/NetworkManager-dns-vip-prepender-worker.yaml

```diff
@@ -0,0 +1,45 @@
+filesystem: "root"
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/dns-vip-prepender-worker"
+contents:
+  inline: |
+    #!/bin/bash
+
+    IFACE=$1
+    STATUS=$2
+
+    case "$STATUS" in
+        pre-up)
+        logger -s "NM dns-vip-prepender-worker triggered by pre-upping ${1}."
+        DNS_VIP="127.0.0.1"
```

Comment:
This IP should be from runtime. We used to have some python script to obtain it.

Reply:
You actually have the script included below, but are not calling it :-)

Reply:
I think we need can just get rid of the prependers anyway.